content/en/docs/concepts/storage/volumes.md

```diff
@@ -543,6 +543,63 @@ spec:
       type: FileOrCreate
 ```
 
+### image
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+An `image` volume source represents an OCI object (a container image or
+artifact) pulled and mounted on the kubelet's host machine.
+
+One example to use the `image` volume source is:
+
+{{% code_sample file="pods/image-volumes.yaml" %}}
+
+The volume is resolved at pod startup depending on which `pullPolicy` value is
+provided:
+
+- `Always`: the kubelet always attempts to pull the reference. Container
+  creation will fail If the pull fails.
+- `Never`: the kubelet never pulls the reference and only uses a local image or
+  artifact. Container creation will fail if the reference isn't present.
+- `IfNotPresent`: the kubelet pulls if the reference isn't already present on
+  disk. Container creation will fail if the reference isn't present and the pull
+  fails.
+
+The volume gets re-resolved if the pod gets deleted and recreated, which means
+that new remote content will become available on pod recreation. A failure to
+resolve or pull the image during pod startup will block containers from starting
+and may add significant latency. Failures will be retried using normal volume
+backoff and will be reported on the pod reason and message. The types of objects
+that may be mounted by this volume are defined by the container runtime
+implementation on a host machine and at minimum must include all valid types
+supported by the container image field. The OCI object gets mounted in a single
+directory (`spec.containers[*].volumeMounts.mountPath`) by merging the manifest
+layers in the same way as for container images. The volume will be mounted
+read-only (`ro`) and non-executable files (`noexec`). Sub path mounts for
```

Comment:
We have updated the list of mount options, I wonder if it makes sense to update these here too?

Reply:
Hm, no that's not a hard requirement.

Reply:
What I mean... the implementation is different from what is says here. If that's not a problem, then no worries. Just caught my eye while reading things.

Reply:
Yes, CRI-O uses slightly higher security standards here, but that's not a requirement from the KEP.

Reply:
OK. I see it. CRI is free to implement more atop the minimum requirement. No worries. Nothing to see here then.